server/controllers/healthController.ts

```
@@ -79,29 +85,64 @@ export const getTargetHistory = async (req: AuthRequest, res: Response) => {
         today.setHours(0, 0, 0, 0);
-
-        const summary = await ActivitySummary.findOneAndUpdate(
-            { userId, date: today },
-            { steps, water, calories, workoutMinutes },
-            { new: true, upsert: true }
-        );
-
-        res.json(summary);
+        const existingSummary = await ActivitySummary.findOne({ userId, date: today });
+
+        let updatedSummary;
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)